daisy_workflows/image_import/suse/translate.py

```diff
@@ -26,67 +26,159 @@
 import utils.diskutils as diskutils
 
 
-network = '''
-BOOTPROTO='dhcp'
-STARTMODE='auto'
-DHCLIENT_SET_HOSTNAME='yes'
-'''
+class Package:
+  name = ''
+  gce = False
+  required = False
+
+  def __init__(self, name, gce, required):
+    self.name = name
+    self.gce = gce
+    self.required = required
+
+
+class SuseRelease:
+  flavor = ''
+  major = ''
+  minor = ''
+  packages = None
+  subscriptions = None
+
+  def __init__(self, flavor, major, minor, packages, subscriptions=None):
+    self.flavor = flavor
+    self.major = major
+    self.minor = minor
+    self.packages = packages
+    self.subscriptions = subscriptions
+
+
+cloud_init = Package('cloud-init', gce=False, required=False)
+gcp_sdk = Package('google-cloud-sdk', gce=True, required=False)
+gce_init = Package('google-compute-engine-init', gce=True, required=True)
+gce_oslogin = Package('google-compute-engine-oslogin', gce=True, required=True)
+
+_distros = [
+    SuseRelease(
+        flavor='opensuse',
+        major='15',
+        minor='1',
+        packages=[cloud_init, gcp_sdk, gce_init, gce_oslogin]
```

Comment:
We shouldn't need cloud-init... in fact it will likely cause conflicts.

Reply:
Interesting .. what kind of conflicts?

This test motivated me to add it: https://github.com/GoogleCloudPlatform/compute-image-tools/blob/06b4dc2fa750de2c40fd5ec37f96c01c5e297371/daisy_integration_tests/scripts/post_translate_test.sh#L142-L141